examples/IterationsTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+use Eris\Generator;
+use Eris\TestTrait;
+
+class IterationsTest extends PHPUnit_Framework_TestCase
+{
+    use TestTrait;
+
+    public function testNumberOfIterationsCanBeConfigured()
+    {
+        $this->setIterations(5)
```

Comment:
Something like `runFor(x)` and the you can pass something that it's going to limit the number of iterations, tomorrow could be a limitation based on time. Another name could be `limitTo(x)`